soccer/gameplay/tactics/stopped/circle_near_ball.py

```diff
@@ -131,7 +131,10 @@ def execute_completed(self):
                 num_robots += 1
 
         i = 0
-        for pt in self.get_circle_points(num_robots):
+        for pt in self.get_circle_points(6 if num_robots == 0 else num_robots):
```

Comment:
I'm kind of confused why the `6 if num_robots == 0 else num_robots` is needed? Could you explain why?